packages/smooth_app/lib/pages/preferences/user_preferences_account.dart

```diff
@@ -298,8 +300,35 @@ class _UserPreferencesPageState extends State<UserPreferencesSection> {
         _getListTile(
           appLocalizations.account_delete,
           () async {
+            await showDialog<SmoothAlertDialog>(
+                context: context,
+                builder: (BuildContext context) {
+                  return SmoothAlertDialog(
+                    title: appLocalizations.account_delete,
+                    body: Column(
+                      mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                      crossAxisAlignment: CrossAxisAlignment.center,
+                      children: <Widget>[
+                        Text(appLocalizations.account_delete_message),
+                        const SizedBox(
+                          height: 5,
+                        ),
+                        SmoothTextFormField(
```

Comment:
Sorry to say that only now, but I don't think a one-line text field is appropriate to give the reason why you want to have your account deleted.
Given that we don't send emails but just open a new email on a mail app with pre-populated fields, I guess we would be better off pre-populating the email body with "Please give us the reason why you want your account to be deleted: ...", and let the user write inside the mail app (instead of a one-line text field).
Just my 2 cents. We can try with that one-line text-field, it's not such a common use case (at least I hope).

Reply:
What needs to be done on my side ? Is the issue closed

Reply:
I agree with you @monsieurtanuki , we open the email app anyways,
What you suggested makes more sense

>What needs to be done on my side ? Is the issue closed

Perhaps we will need other pr with the changes that is requested here ,

It's more like a ux problem , 
@M123-dev  @teolemon

Reply:
@AshAman999 That's why we should exchange about issues at the issues level ("Which solutions make sense?"), and then exchange about the code at the PRs level.